bundle/pom.xml

```diff
@@ -36,13 +36,14 @@
 								<artifactItem>
 									<groupId>com.jfrog.ide</groupId>
 									<artifactId>ide-plugins-common</artifactId>
-									<version>1.0.0</version>
+									<version>1.1.0</version>
```

Comment:
Since the *ide-plugins-common* version is used twice in the pom, I suggest using a variable (property for it).